src/main/java/net/imglib2/outofbounds/OutOfBoundsConstantValue.java

```diff
@@ -85,4 +97,12 @@ final public T get()
 	{
 		return copy();
 	}
+
+	private static < T > Supplier< T > makeSupplierFrom( final T t ) {
+		if ( t instanceof Type< ? > ) {
```

Comment:
I would make this `t instanceof Type` (instead of `Type< ? >`)
It's kind of an edge case, but for example `t instanceof Type< ConcreteType >` doesn't compile, because instanceof doesn't check generic parameters.

Reply:
Should I update the cast to raw type as well (`( Type )` instead of `( Type< ? > )`?

Reply:
I think the cast should still be to `Type< ? >` because that's the properly specified generic type here. You don't know anything about the bound; writing `Type< ? >` is the most general expression for that without introducing type variables.